api/src/main/java/com/github/retrooper/packetevents/util/adventure/AdventureNBTSerializer.java

```diff
@@ -1,3 +1,21 @@
+/*
+ * This file is part of ProtocolSupport - https://github.com/ProtocolSupport/ProtocolSupport
+ * Copyright (C) 2021 ProtocolSupport
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
```

Comment:
Are you sure this is ProtocolSupport? :smile:

Reply:
I copied it from NBT class file. My bad